### ISSUE TITLE
Repeating Events, infinite, stopped

### ISSUE THREAD

**Issue by michaelof** [SimpleMobileTools/Simple-Calendar#901]
Simple-Calendar, current F-Droid 6.5.0
Android 6.0
DAVx5 2.5.0-ose
Nextcloud, current 16.0.1
(iOS 12.2)

Hi Tibor,

issue with infinite (repeat forever, once  a week) events: 
- created infinite at first
- after some months, I've deleted "this and future occurences"

Result:
- Event correctly shown in SimpleCalendar, (stopped after specific date)
- Event shown correctly in Nextcloud. Although Nextcloud says "Event's repeating rule is not supported yet"
- Event is NOT shown correctly in iOS 12.2, not stopped, still infinite.

**Comment by tibbi** [comment]
well I dont see why would it be an issue on our side, if it works fine in the Android app and NextCloud. Does it show well also in other android apps?

**Comment by michaelof** [comment]
Wow, while crosschecking because of your question I've found out that I've to correct/"invert" my issue:

ONLY iOS seems to be correct:

As said, after some months, I've deleted "this and future occurences". I fact I did this for the event occurence of TODAY.

Looking into SimpleCalendar, I see that event is shown as "repeat every Tuesday UNTIL 28.5.2019".

So the difference between iOS, Nextcloud and Android is, that iOS treats "repeat until" as "repeat until INCLUDING the 'until' date" whereas all others treat the until date to be excluded. 

IMHO iOS is correct here, for my understanding of "repeat until date" :) But as all other calendars treat it the same way, fine for me. Especially as SimpleCalendar did exactly, what I've wanted, "delete this and future.."

Important for me is that from next Tuesday on the event also has stopped in iOS. One instance too much is OK :)
